.github/workflows/production.yml

```diff
@@ -1,112 +0,0 @@
----  
```

Comment:
I don't think this release process (or the release to QA) should be removed until we've actually done a few releases through to production with the new system.

This can get removed in a future PR

Reply:
Manual workflow is there for that reason, it can release to Test or Production ad-hoc